CooperationHullMainSite/Views/Home/Index.cshtml

```diff
@@ -90,56 +90,49 @@
 </section>
 
 
-<section class="homepage-block sign-up">
-    <div class="row">
-
-        <div class="col-lg-6 col-md-12 form-container order-lg-0 order-md-1">
-
-            <div id="homepage-form-to-complete">
-                <p id="homepage-formError"></p>
-                <form method="post" id="home_page_signup_form" asp-action="Home_page_signup_form" asp-controller="Home">
-                    <div class="row m-3">
-                        <div class="col-xl-6 col-lg-12">
-                            <label for="PledgeFormFirstName" class="form-label">First Name</label>
-                            <input id="PledgeFormFirstName" name="PledgeFormFirstName" class="form-control form-control-lg" type="text" placeholder="First Name" aria-label="First Name">
-                        </div>
-                        <div class="col-xl-6 col-lg-12">
-                            <label for="PledgeFormSurname" class="form-label">Surname</label>
-                            <input id="PledgeFormSurname" name="PledgeFormSurname" class="form-control form-control-lg" type="text" placeholder="Surname" aria-label="Surname">
-                        </div>
+<section class="homepage-block call-out-block-wrapper">
+    <div class="call-out-block">
+        <div class="row">
+            <div class="col-lg-6">
+                <h2 class="heading-md">Want to be part of something bigger?</h2>
+                <p>
+                    If you want to make the world a better place, leaving us your phone number is the easiest way to make a difference. In a short call we’ll answer your questions and find your place in the movement. Join hundreds of local people, working together, today.
+                </p>
+                <div class="p-4 order-lg-1 order-md-0">
+                    @if (Model.formSignedCounter >= 0)
+                    {
+                        <p>@($"{Model.formSignedCounter} have signed so far")</p>
+                    }
+                    else
+                    {
+                        <p>Hundreds of people have signed already</p>
+                    }
+                </div>
+            </div>
+            <form class="col-lg-6" method="post" id="home_page_signup_form" asp-action="Home_page_signup_form" asp-controller="Home">
```

Comment:
Related to https://github.com/Fireflower2012/CooperationHullMainSite/pull/83#discussion_r1600608559, @Fireflower2012 do you think we'll be able to write our own form here, or will we have to use an embed as well?

Reply:
We're already using our own working code for the submission on the home page.
Long story but it's not actually an action network form as their API is annoying but a person sign up helper which they count as something different to a form for reasons that must make sense to them!